lib/collection/item.js

```diff
@@ -329,7 +329,28 @@ _.assign(Item.prototype, /** @lends Item.prototype */ {
         if (!this.request) { this.request = new Request(); } // worst case
 
         return this.request.authorizeUsing(type, options);
+    },
+
+    /**
+     * Returns the path of the item
+     *
+     * @returns {Array<string>}
+     */
+    getPath: function () {
+        const path = [];
+        let item = this;
+
+        while (item) {
+            if (typeof item.name === 'string') {
```

Comment:
Let's check why the `if` is needed here.

Reply:
Added comment